base_tier_validation_forward/models/tier_review.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class TierReview(models.Model):
+    _inherit = "tier.review"
+    _order = "sequence"
+
+    name = fields.Char(compute="_compute_definition_data", store=True, related=False)
```

Comment:
We spotted a bug when we tried to install this 18.0 module to a customer.
To fix the bug, here we need a translate=True as well.

```suggestion
    name = fields.Char(compute="_compute_definition_data", store=True, related=False, translate=True)
```

Explanation:
In `base_tier_validation` field name is a related field and is `translate=True`. But here the related field is replaced with a computed field. This seems causing the loosing the translate=True setting for this field.
 
In `_compute_definition_data(...)` we have this line:
  `rec.name = rec.definition_id.name`
 
In case multiple languages are installed, you will get the following error:

```
psycopg2.errors.UndefinedFunction: function jsonb_path_query_first(character varying, unknown) does not exist
LINE 4: 'en_US', jsonb_path_query_fi...
^
```

The reason is that the jsonb string in rec.definition_id.name is assigned to rec.name which is a varchar.

Not sure if the explanation is clear, just let me know if you need more info.

Reply:
@astirpe I can't reproduce to error. Can you help steps to reproduct please?

Reply:
@Saran440 
I tried to reproduce it locally but I didn't get the error this time. I was only getting the error on a customer environment, don't know why. Sorry.
However it's already fixed now by putting translate=True on the field definition.

Besides of the error, I still think that translate=True is needed here.
And also, in the compute method, instead of this line:
`    rec.name = rec.definition_id.name`
we need a deepcopy of the definition_id.name, so that all the translations will be copied.

Reply:
@astirpe I added translate=True on name field. please re-check.

Reply:
Thank you!